samples/Calculator/cppwinrt/README.md

```diff
@@ -27,26 +27,28 @@ To upgrade this sample to the latest version of RNW:
     ```cmd
     npx @react-native-community/cli@latest init Calculator --template @react-native-community/template@latest
     ```
-4. Add Windows support:
+4. Add `yarn.lock` file to app directory (otherwise `yarn add react-native-windows@latest` will throw an error)
+
+5. Add Windows support:
     ```cmd
     cd Calculator
     yarn add react-native-windows@latest
```

Comment:
You might want to add a comment here about using the version range instead of the word `latest`.